board/drivers/harness.h

```diff
@@ -39,7 +39,7 @@ void set_intercept_relay(bool intercept, bool ignition_relay) {
     }
 
     // wait until we're not reading the analog voltages anymore
-    while (harness.sbu_adc_lock == true) {}
+    while ((bool)harness.sbu_adc_lock == true) {}
```

Comment:
`harness.sbu_adc_lock` is already a bool, I suspect it's complaining about the "== true" that you can just remove. it's really important to do these fixes in a principled way and to think very carefully about them